extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -153,6 +155,7 @@ public void build(RecorderContext recorderContext, HibernateOrmRecorder recorder
 
         if (!hasEntities(domainObjects, nonJpaModelBuildItems)) {
             // we can bail out early
+            log.warn("No entities were found!");
```

Comment:
I'm not a big fan of this warning as you will have it as soon as you add the ORM extension. I would remove it.